README.md

```diff
@@ -54,275 +94,43 @@ See the [examples](https://github.com/GoogleCloudPlatform/gcp-service-broker/tre
-             "instance_name": "redacted",
-             "last_master_operation_id": "some-guid",
-             "uri": "mysql://username:encodedpassword@host/databasename?ssl_mode=required"
-        }
-        ```
+## Usage
 
-* [ML APIs](https://cloud.google.com/ml/)
-    * Bind
-        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles)
-
-        **Example Binding credentials**
-
-        ```json
-        "credentials": {
-             "Email": "redacted",
-             "Name": "redacted",
-             "PrivateKeyData": "redacted",
-             "UniqueId": "redacted",
-        }
-        ```
-
-* [Bigtable](https://cloud.google.com/bigtable/docs/)
-    * Provison
-        * `name` (defaults to a generated value)
-        * `cluster_id` (defaults to a generated value)
-        * `display_name` (defaults to a generated value)
-        * `storage_type` (one of "SSD" or "HDD", defaults to "SSD")
-        * `zone` (defaults to us-east1-b)
-        * `num_nodes` (defaults to 3)
-    * Bind
-        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles), e.g. editor
-
-        **Example Binding credentials**
-
-        ```json
-        "credentials": {
-             "Email": "redacted",
-             "Name": "redacted",
-             "PrivateKeyData": "redacted",
-             "UniqueId": "redacted",
-             "instance_id": "foobar",
-        }
-        ```
-
-* [Spanner](https://cloud.google.com/spanner/docs/) (BETA Google Service)
-    * Provison
-        * `name` (defaults to a generated value)
-        * `display_name` (defaults to a generated value)
-        * `location` (defaults to regional-us-central1)
-    * Bind
-        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles), e.g. spanner.admin
-
-        **Example Binding credentials**
-
-        ```json
-        "credentials": {
-             "Email": "redacted",
-             "Name": "redacted",
-             "PrivateKeyData": "redacted",
-             "UniqueId": "redacted",
-             "instance_id": "foobar",
-        }
-        ```
+See [here](https://github.com/GoogleCloudPlatform/gcp-service-broker/blob/master/docs/use.md) for instructions on creating and binding to GCP Services
+ 
+See the [examples](https://github.com/GoogleCloudPlatform/gcp-service-broker/blob/master/examples/) folder to understand how to use services once they are created and bound.
 
-* [Stackdriver Debugger](https://cloud.google.com/debugger/)
-    * Provison (none)
-    * Bind (none)
-	* provided credentials will have the role of `clouddebugger.agent
+## Commands
+
+The [cmd](insert link) folder contains commands that can be run independent of the broker.
+
+* `migrate`: migrates the database to the latest schema
+
+## Testing
+
+Production testing for the GCP Service Broker is administered via a private Concourse pipeline.
```

Comment:
Here's a guide: http://engineering.pivotal.io/post/concourse-badges/
The badge is at https://concourse-server-url/api/v1/pipelines/pipeline-name/jobs/job-name/badge